django/librarian/migrations/0007_remove_library_chat_datasource_chat.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.1 on 2024-09-19 15:32
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("chat", "0014_chatoptions_qa_documents_chatoptions_qa_scope"),
+        ("librarian", "0006_alter_library_options_library_is_personal_library"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="library",
+            name="chat",
+        ),
+        migrations.AddField(
+            model_name="datasource",
+            name="chat",
+            field=models.OneToOneField(
```

Comment:
This is OK for now but I generally don't like that DataSource has a chat attribute. Would rather the link be the other way or as a joining table.

Reply:
That makes sense, since every chat will have a data source but not every data source has a chat.
In practice though, doesn't make a difference since you can lookup from the chat side by related name.
Will see about changing it.